docs/configuration/config-file-reference.md

```diff
@@ -126,6 +126,11 @@ api:
 # Cortex query-frontend.
 [query_range: <query_range_config>]
 
+query:
+  # Number of shards to use when distributing shardable PromQL queries.
+  # CLI flag: -querier.vertical-shard-size
```

Comment:
it's weird that the flag is `querier` when the config is under `query` section

Reply:
This is indeed a question that i dont know what is the best solution.

I added as `query` as the other configurations are under `query_range`... So i just followed the same pattern.

WDYT?

Reply:
🤔 yea I think it's confusing too, but yea I guess may be just follow existing convention for now.